### ISSUE TITLE
checkpoint discriminator

### ISSUE THREAD

**Issue by hyolee1999** [zengxianyu/crfill#3]
Can you provide the checkpoint for discriminator?

**Comment by zengxianyu** [comment]
I'll add discriminator later

**Comment by hyolee1999** [comment]
I'm have been retraining your model recently ,  and because you have not published the training code yes so i've implemented it based on provided code and your paper . But still , there are some parts i've not fully understood , hoping you can help me out:

1. In the Discriminator code , you create a full one matrix and concatenate it with the input image but in Deepfillv2 , they concatenate the input with mask , so which one is right?

2.In the CP1 module , you use hardmax or softmax ? fuse patchs or not ? because I trained it with not fusing and it generates bad when the mask is large.

3.When training model to work on high resolution , you use image size  512x512 or 256x256 . Does it work when I use 256x256 size for training?.

Thank you.

**Comment by zengxianyu** [comment]
the training code is released
